message.go

```diff
@@ -79,7 +79,7 @@ type messageRequestMessage struct {
 	MentionedUsers []string               `json:"mentioned_users"`
 	ParentID       string                 `json:"parent_id"`
 	ShowInChannel  bool                   `json:"show_in_channel"`
-	ExtraData      map[string]interface{} `json:"-,extra"`
+	ExtraData      map[string]interface{} `json:"-,extra"` //nolint: staticcheck
```

Comment:
Could you please provide a link to the Jira ticket related to this change? I'm in doubt because we have `message` type on the backend with a field called `custom`, not `extra`.